cesql/spec.md

```diff
@@ -314,8 +313,6 @@ For example, the following definitions are valid:
 * `ABC(x, y): String x String -> Integer`: Binary function (arity 2).
 * `ABC(x, y, z, ...): String x String x String x String^n -> Integer`: n-ary function (variable arity), but the initial fixed arguments are at least 3.
 
-But the followings are invalid, so the engine MUST reject them:
```

Comment:
@drewpca why did you remove this line?

Reply:
Looks like a merge error. I see no reason to remove that line

Reply:
ok thanks - I'll add it back in